lib/govuk_test.rb

```diff
@@ -11,10 +11,16 @@ def self.configure(options = {})
     chrome_options << "--window-size=#{options[:window_size]}" if options[:window_size]
 
     if ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']
-      # Use the installed chromedriver, rather than chromedriver-helper
+      # Use the installed chromedriver, rather than webdrivers
       Selenium::WebDriver::Chrome.driver_path = File.which("chromedriver")
     else
-      require 'chromedriver-helper'
+      require 'webdrivers'
+      if Gem.loaded_specs.has_key?('webmock')
```

Comment:
I wonder if we should do `if defined?(WebMock)` here instead just in case `WebMock` is available from somewhere other than the Gem (even though this is unlikely). I notice this is commonly used in govuk_app_config: https://github.com/alphagov/govuk_app_config/search?q=defined&unscoped_q=defined